libmattermost.c

```diff
@@ -4206,6 +4237,9 @@ PurpleConnection *pc
 	act = purple_protocol_action_new(_("Search for friends..."), mm_search_users);
 	m = g_list_append(m, act);
 
+	act = purple_protocol_action_new(_("Room List"), mm_roomlist_show);

```

Comment:
Why have you added in this account option, rather than using the standard Tools->Room List?